config/samples/mig-migration.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
 spec:
   # [!] Set 'cancel: true' will cancel the migration
   cancel: false
```

Comment:
@djwhatle Just out of curiosity, shouldn't there be ```canceled```?

Reply:
You're correct :) we need an update to migsamples.